tools/run

```diff
@@ -70,14 +59,16 @@ async function withCwd(dir, fn) {
 }
 
 function testWasm() {
-    let workingDir = path.dirname(__filename)
-    let sourceDir = path.join(workingDir, '../lib/rust')
+    let rootDir = path.join(path.dirname(__filename), '..')
 
-    dirs = fss.readdirSync(sourceDir, {withFileTypes: true})
-        .filter(f => f.isDirectory())
-        .map(d => path.join(sourceDir, d.name))
+    content = fss.readFileSync(path.join(rootDir, 'Cargo.toml'), {encoding: 'utf-8'})
+    content = content.replace(/\s+|"/g, '')
+    members = content.split('members')[1].split('[')[1].split(']')[0]
+    dirs    = members.split(',').map(d => path.join(rootDir, d))
 
-    dirs.forEach(d => withCwd(d, testLibrary))
+    dirs.forEach(d => {
```

Comment:
This also _doesn't_ test `enso-macro-utils`, which should be tested. It's not a proc macro crate. The condition you want is that the folder you enter contains both a `macros` and an `impl` folder, and you only want to test in the `impl` folder.